lighthouse-core/test/lib/traces/tracing-processor-test.js

```diff
@@ -159,4 +159,62 @@ describe('TracingProcessor lib', () => {
       assert.deepEqual(results, expected);
     });
   });
+
+  describe('log normal distribution', () => {
+    it('creates a log normal distribution', () => {
+      // This curve plotted with the below percentile assertions
+      // https://www.desmos.com/calculator/vjk2rwd17y
+
+      const median = 5000;
+      const pODM = 3500;
+      const distribution = TracingProcessor.getLogNormalDistribution(median, pODM);
+
+      function getPct(distribution, value) {
+        return distribution.computeComplementaryPercentile(value).toFixed(2);
+      }
+      assert.equal(typeof distribution.computeComplementaryPercentile, 'function');
+      assert.equal(getPct(distribution, 2000), '1.00', 'pct for 2000 does not match');
+      assert.equal(getPct(distribution, 3000), '0.98', 'pct for 3000 does not match');
+      assert.equal(getPct(distribution, 3500), '0.92', 'pct for 3500 does not match');
+      assert.equal(getPct(distribution, 4000), '0.81', 'pct for 4000 does not match');
+      assert.equal(getPct(distribution, 5000), '0.50', 'pct for 5000 does not match');
+      assert.equal(getPct(distribution, 6000), '0.24', 'pct for 6000 does not match');
+      assert.equal(getPct(distribution, 7000), '0.09', 'pct for 7000 does not match');
+      assert.equal(getPct(distribution, 8000), '0.03', 'pct for 8000 does not match');
+      assert.equal(getPct(distribution, 9000), '0.01', 'pct for 9000 does not match');
+      assert.equal(getPct(distribution, 10000), '0.00', 'pct for 10000 does not match');
+    });
+  });
+
+  describe('risk to responsiveness', () => {
+    let oldFn;
+    // monkeypatch _riskPercentiles to deal with gRtR solo
+    beforeEach(() => {
+      oldFn = TracingProcessor._riskPercentiles;
+      TracingProcessor._riskPercentiles = (durations, totalTime, percentiles, clippedLength) => {
+        return {
+          durations, totalTime, percentiles, clippedLength
+        };
+      };
+    });
+    afterEach(() => {
+      TracingProcessor._riskPercentiles = oldFn;
+    });
+
+    it('gets durations of top-level tasks', () => {
```

Comment:
for this test, I think it would be better to just pull out a "get top level slices from main thread" function in `tracing-processor.js` and test that